src/main/java/com/wavefront/opentracing/WavefrontSpan.java

```diff
@@ -38,6 +41,9 @@
   private boolean finished = false;
   private boolean isError = false;
 
+  // Store it as a member variable so that we can efficiently retrieve the component tag.
+  private String componentTagValue = NULL_TAG_VAL;
```

Comment:
Is it ok to report NULL tag for component value? can we not include the component tag when it null?